routes/account.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const router =  express.Router()
+
+const { isAuthorised } = require('../middleware/auth')
+const { locals } = require('../utils/format')
+
+router.get('/index', isAuthorised, (req, res) => {
+    res.render('account/index', req.session?.user)
+})
+
+router.get('/list', isAuthorised, (req, res) => {
+    res.render('account/list', req.session?.user)
+})
+
+router.get('/logout', isAuthorised, (req, res) => {
```

Comment:
suggestion: I'd keep the auth endpoints together until a single router - login, register, logout, perhaps under `routes/auth.js`